core/deployment/src/test/java/io/quarkiverse/operatorsdk/test/UnownedTest.java

```diff
@@ -0,0 +1,38 @@
+package io.quarkiverse.operatorsdk.test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.fabric8.kubernetes.client.CustomResource;
+import io.quarkiverse.operatorsdk.test.sources.*;
+import io.quarkus.test.ProdBuildResults;
+import io.quarkus.test.ProdModeTestResults;
+import io.quarkus.test.QuarkusProdModeTest;
+
+public class UnownedTest {
+    // Start unit test with your extension loaded
+    @RegisterExtension
+    static final QuarkusProdModeTest config = new QuarkusProdModeTest()
+            .setApplicationName(UnownedReconciler.NAME)
+            .withApplicationRoot(
+                    (jar) -> jar.addClasses(UnownedReconciler.class, TestCR.class))
+            .overrideConfigKey("quarkus.operator-sdk.crd.generate-all", "true")
+            .overrideConfigKey("quarkus.operator-sdk.controllers." + UnownedReconciler.NAME + ".unowned-primary", "true");
```

Comment:
This is the salient part of this test: setting the `unowned-primary` property to `true` for the controller in application.properties.